cosmo_tester/resources/terraform/scripts/osx-cli-test.sh

```diff
@@ -18,20 +14,20 @@ echo $MACINCLOUD_PASSWORD | sudo -S installer -pkg /tmp/package.pkg -target /
 
 set -e
 
-echo "Creating inputs file.."
-echo "public_ip: ${PUBLIC_IP}
-private_ip: ${PRIVATE_IP}
-ssh_user: ${MANAGER_USER}
-ssh_key_filename: ${PRIVATE_KEY_PATH}
-admin_username: admin
-admin_password: admin" > ${INPUTS_FILE_PATH}
-cat ${INPUTS_FILE_PATH}
+cfy=/usr/local/opt/cfy/bin/cfy
```

Comment:
Given that this is the largely the same as the other code, is it possible for us to put this in another file and source it (or build the script from a jinja template perhaps?).

Reply:
I considered it, but you need a separate script anyway, for platform specific stuff. You _could_ have a 3rd common script which both linux and OSX would source, but I also feel like it would warrant its own PR.